packages/crayons-core/src/components/avatar/readme.md

```diff
@@ -89,7 +89,7 @@ function App() {
 | `alt`      | `alt`      |             | `string`                                                                           | `undefined` |
 | `image`    | `image`    |             | `string`                                                                           | `undefined` |
 | `initials` | `initials` |             | `string`                                                                           | `undefined` |
-| `mode`     | `mode`     |             | `"dark" \| "light"`                                                                | `'dark'`    |
+| `mode`     | `mode`     |             | `"dark" \| "error" \| "light"`                                                     | `'dark'`    |
```

Comment:
please update this readme file with samples to show the newly added mode and other variations